### ISSUE TITLE
Automatically update IDE settings according to project data

### ISSUE THREAD

**Issue by ivankravets** [platformio/platformio#364]
In the current state PlatformIO initializes project for the specified IDE and board using existing source code. When user modifies project and uses new libraries, then the will not be reflected in IDE settings. The only one solution is to reinitialize IDE settings.

See related isse #362

**Comment by MacDada** [comment]
Yep, I'm confused _when_ to run it manually now… https://community.platformio.org/t/clion-is-re-init-the-same-as-pio-init-ide-clion-when-to-use-it/31882?u=dvdnwk
